github/reactions_test.go

```diff
@@ -184,6 +184,92 @@ func TestReactionsService_CreatePullRequestCommentReaction(t *testing.T) {
 	}
 }
 
+func TestReactionsService_ListTeamDiscussionReactions(t *testing.T) {
+	client, mux, _, teardown := setup()
+	defer teardown()
+
+	mux.HandleFunc("/teams/1/discussions/1/reactions", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
This is a tiny nit, but here and below, please make the parameters unique for two reasons:
* unique parameters make sure that the order and matching of name to value is correct, and
* it makes it easier to see which parameter goes where and affects which values for someone quickly glancing at the code.

So here and below, if you have two or more values that are the same (e.g. `1`), make then all unique.